requirements/base-requirements.in

```diff
@@ -95,7 +96,7 @@ requests_toolbelt
 requests
 schema
 sentry-sdk
-sh
```

Comment:
Is this intentional?

Reply:
No, I am not sure how that got removed. I'll do a commit so that this library is back in as no libraries should be removed.